src/run/FshToFhir.ts

```diff
@@ -104,6 +106,7 @@ type fshToFhirOptions = {
   fhirVersion?: string;
   dependencies?: ImplementationGuideDependsOn[];
   logLevel?: Level;
+  snapshot?: boolean;
```

Comment:
Do you think it might be worth putting a comment here that warns that SUSHI snapshot generation is experimental (or some word like that) and that snapshots may differ from what the IG Publisher would generate? I just worry because snapshots are "good enough" for our own purposes within SUSHI (and likely for some other use cases), but they're not really a focus of the tool -- so I think maybe people should know that?

Reply:
I'm noticing now that we don't say anything about this in any of our documentation elsewhere (like the FSH School documentation) -- so maybe I'm being overly cautious. Feel free to push back on this if you'd like.

Reply:
Valid points! I'll add a comment to that affect. I'll also add a comment in the `snapshot` parameter in the FSH School doc update. After I resubmit bot PRs, let me know if you want me to do any additional word-smithing.